spec/controllers/categories_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+require 'support/app_controller'
+
+
+RSpec.describe CategoriesController, type: :controller do
+
+  let(:create_params) do {
+    name: 'Category'
+  }
+
+  end
+
+  let (:show_item) {Category.create(create_params)}
```

Comment:
The `let` block on lines 7 - 11 implies that `create_params` will be used for specific tests. It might make it easier to read if you combined the two above to `let` blocks.